pubtools/pulplib/_impl/client/repo_lock.py

```diff
@@ -0,0 +1,161 @@
+import json
+import random
+import string
+from time import sleep
+
+from datetime import datetime, timedelta
+from more_executors.futures import f_map
+
+
+LOCK_CLAIM_STR = "lock-claim-"
```

Comment:
It should probably start with "pulplib-" so it's possible to trace these back to the library which created them, and also avoid clashes with anything else.